src/platform/user/profile/vet360/components/AddressField/AddressField.jsx

```diff
@@ -135,6 +157,224 @@ export const convertCleanDataToPayload = (cleanData, fieldName) =>
     },
     e => !!e,
   );
+};
+
+const formSchema = {
+  type: 'object',
+  properties: {
+    'view:livesOnMilitaryBase': {
+      type: 'boolean',
+    },
+    'view:livesOnMilitaryBaseInfo': {
+      type: 'object',
+      properties: {},
+    },
+    countryName: {
+      type: 'string',
+      enum: ADDRESS_FORM_VALUES.COUNTRIES,
+    },
+    addressLine1: {
+      type: 'string',
+      minLength: 1,
+      maxLength: 100,
+      pattern: '^.*\\S.*',
+    },
+    addressLine2: {
+      type: 'string',
+      minLength: 1,
+      maxLength: 100,
+      pattern: '^.*\\S.*',
+    },
+    addressLine3: {
+      type: 'string',
+      minLength: 1,
+      maxLength: 100,
+      pattern: '^.*\\S.*',
+    },
+    city: {
+      type: 'string',
+    },
+    stateCode: {
+      type: 'string',
+      enum: Object.keys(ADDRESS_DATA.states),
+      enumNames: Object.values(ADDRESS_DATA.states),
+    },
+    province: {
+      type: 'string',
+    },
+    zipCode: {
+      type: 'string',
+      pattern: '^\\d{5}$',
+    },
+    internationalPostalCode: {
+      type: 'string',
+    },
+  },
+  required: ['countryName', 'addressLine1', 'city'],
+};
+
+const uiSchema = {
```

Comment:
I might recommend pulling these out into a separate file just because of the added length.